newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -324,6 +325,8 @@ export type Preferences = {|
     projectId: string,
     editorState?: {| editorTabs: EditorTabsPersistedState |}
   ) => void,
+  getFetchPlayerTokenForPreviewAutomatically: () => boolean,
```

Comment:
I don't think you need the getter here since you directly use the preference value